package.json

```
@@ -42,6 +42,7 @@
         "setup": "pnpm install",
         "start": "pnpm build --stream --filter coveo-styleguide && pnpm -r start --parallel --filter coveo-styleguide --filter react-vapor-demo",
         "test": "pnpm -r test --parallel",
+        "test:ci": "pnpm -r test:ci --parallel",
```

Would it be possible to put the long options instead of the shortcut. Easier to understand what the order wants to do when you start and it also helps to improve maintenance :smile: 

```suggestion
        "test:ci": "pnpm --recursive test:ci --parallel",
```

---

notons ceci au calendrier. Je suis completement d'accord avec Yann sur le coup ! :heart: :wink: